views/styles/account_page_styles.js

```diff
@@ -0,0 +1,33 @@
+import { colors } from '../../constants'
+
+export const accountPageStyles = (theme) => ({
+  form: {
+    padding: '12px',
+    overflow: 'scroll',
```

Comment:
I believe this keeps the scroll bar on no matter what, is that what you want?

Reply:
good catch no we don't [0ca9ade](https://github.com/AMP-SCZ/dpdash/pull/320/commits/0ca9ade589cf737b00589944549c08cd5fa728bc)